src/ML-POD/compute_podfit.h

```diff
@@ -0,0 +1,214 @@
+/***************************************************************************
+               CESMIX-MIT Project
+
+ Contributing authors: Ngoc-Cuong Nguyen (cuongng@mit.edu, exapde@gmail.com)
+ ***************************************************************************/
+
+#ifdef COMPUTE_CLASS
+// clang-format off
+ComputeStyle(podfit,CPODFIT);
+// clang-format on
+#else
+
+#ifndef LMP_COMPUTE_PODFIT_H
+#define LMP_COMPUTE_PODFIT_H
+
+#include "compute.h"
+#include "pod.h"
+
+namespace LAMMPS_NS {
+
+class CPODFIT : public Compute {
+private:
+
+  class NeighList *list;
+  std::vector<std::string> globVector(const std::string& pattern, std::vector<std::string> & files);
```

Comment:
Convenience functions like this and the following should not be class members. Rather they should be static functions at the top of the implementation, or - if possible - replaced by existing convenience functions in the "utils" and "platform" namespaces of LAMMPS.

Reply:
Function declaration is removed as It is defined as a static function at the top of the file.